pfcwd/main.py

```diff
@@ -194,9 +195,23 @@ def interval(poll_interval):
     configdb.connect()
     pfcwd_info = {}
     if poll_interval is not None:
+        pfcwd_table = configdb.get_table(CONFIG_DB_PFC_WD_TABLE_NAME)
+        entry_min = 3000
+        for entry in pfcwd_table:
+            if("Ethernet" not in entry):
+                continue
+            detection_time_entry_value = int(configdb.get_entry(CONFIG_DB_PFC_WD_TABLE_NAME, entry).get('detection_time'))
+            restoration_time_entry_value = int(configdb.get_entry(CONFIG_DB_PFC_WD_TABLE_NAME, entry).get('restoration_time'))
+            if ((detection_time_entry_value != None) and (detection_time_entry_value < entry_min)):
+                entry_min = detection_time_entry_value
+            if ((restoration_time_entry_value != None) and (restoration_time_entry_value < entry_min)):
+                entry_min = restoration_time_entry_value
+        if entry_min <= poll_interval:
```

Comment:
The condition should be just '<'. A user will never be able to configure the upper bound value which is 3000 because of "=" check. Also most of the current device configs have the same value for detection, restoration and poll interval. We will be restricting that config as well